notebooks/ACS/acs_polarization_tools/acs_polarization_tools.ipynb

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
<p>Hello @alyssaguzman, Welcome to the notebook review cycle for ACS notebooks. As we begin the review process, I would like to mention that this notebook is relatively short, but it is well-written and executed. However, there is a minor issue that needs to be addressed. The link to the documentation is broken and needs to be updated. Thank you.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
The current working link to the ACS Destripe is ```https://acstools.readthedocs.io/en/latest/acs_destripe.html#module-acstools.acs_destripe```.